05_outputs.Rmd

```diff
@@ -358,6 +358,167 @@ for (i in 1:length(col_names)) {
 
 ### change from label to xml
 
+The following code transform column headers, select_one and select_multiple values from Label to XML.
+
+Conditions required for the code to function correctly:
+1- The "label::English" column in KOBO Tool must contain unique values.
```

Comment:
Can we make it dynamic? Not all label column are always "label::English", maybe we can use label_colname as parameter and provide a default value.

Reply:
Done!